tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/RabbitMQ/BasicPublishIntegration.cs

```diff
@@ -79,7 +79,7 @@ public class BasicPublishIntegration
                     }
 
                     SpanContextPropagator.Instance.Inject(scope.Span.Context, basicProperties.Headers, default(ContextPropagation));
-                    RabbitMQIntegration.SetDataStreamsCheckpointOnProduce(Tracer.Instance, scope.Span, tags, basicProperties.Headers);
+                    RabbitMQIntegration.SetDataStreamsCheckpointOnProduce(Tracer.Instance, scope.Span, tags, basicProperties.Headers, body?.Length ?? 0);
```

Comment:
`body` is a struct, so you need to do 
```suggestion
                    RabbitMQIntegration.SetDataStreamsCheckpointOnProduce(Tracer.Instance, scope.Span, tags, basicProperties.Headers, body.Instance?.Length ?? 0);
```

Reply:
I'll update the code to be inline with the code above `tags.MessageSize = body.Instance != null ? body.Length.ToString() : "0";`